src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -112,6 +112,13 @@ internal static async Task CheckForUpdates()
                 return;
             }
 
+            // Daily builds do not support update notifications
+            if (PSVersionInfo.PSCurrentVersion.PreReleaseLabel != null
+                && PSVersionInfo.PSCurrentVersion.PreReleaseLabel.StartsWith("daily"))
```

Comment:
```suggestion
                && PSVersionInfo.PSCurrentVersion.PreReleaseLabel.StartsWith("daily", StringComparison.OrdinalIgnoreCase))
```

Reply:
daily builds are already lower-cased, if that changes at some point, I'd consider it a bug.

Reply:
Semver labels are case-sensitive